crates/bevy_core/src/time/fixed_timestep.rs

```diff
@@ -83,24 +95,14 @@ impl FixedTimestep {
         }
     }
 
-    pub fn with_label(mut self, label: &str) -> Self {
-        self.state.label = Some(label.to_string());
+    pub fn with_label(mut self, label: impl Into<String>) -> Self {
```

Comment:
this ```impl Into<String>``` change is irrelevant, it saves and extra string clone when label String is created on the fly and user just want to hand over ownership, which is not allowed with a &str parameter